cocos/scripting/js-bindings/script/jsb_cocos2d.js

```diff
@@ -2693,6 +2693,30 @@ _p.unscheduleUpdateForTarget = _p.unscheduleUpdate;
 _p.unscheduleAllCallbacksForTarget = _p.unscheduleAllForTarget;
 
 
+cc._NodeGrid = cc.NodeGrid;
+cc.NodeGrid = function(rect){
+    if (!(this instanceof cc.NodeGrid)){
+        cc.error("NodeGrid Constructor can not called as a function, Please use new");
```

Comment:
NodeGrid's constructor can not be called as a function, please use 'new cc.NodeGrid()'